Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs

```diff
@@ -16,6 +16,9 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+extern alias realm;
```

Comment:
Well, `global`, the default alias, is lowercase, but I don't know of a convention.